source/containers.tex

```diff
@@ -10725,6 +10737,34 @@
 Equivalent to: \tcode{return *(data() + idx);}
 \end{itemdescr}
 
+\indexlibrarymember{span}{front}%
+\begin{itemdecl}
+constexpr reference front() const;
+\end{itemdecl}
```

Comment:
Please add blank line after itemdecl - please do this in all cases.

Reply:
Local consistency?

Reply:
Fixed throughout [views.span].